backends/p4fmt/README.md

```diff
@@ -0,0 +1,24 @@
+# p4fmt (P4 Formatter)
+
+p4fmt is a WIP formatter for P4.
```

Comment:
I would some more detail here that this is not ready yet and not meant for use.

Reply:
Let me know if the updated one looks good!